src/ServerConfigParams.ts

```diff
@@ -307,6 +310,18 @@ export async function resolveServerConfig(
     ) {
         error(`RelayHub: no contract at address ${config.relayHubAddress}`);
     }
+
+    if (config.collectorAddress) {
+        const isCollectorDeployed = await contractInteractor.isContractDeployed(
+            config.collectorAddress
+        );
+        if (!isCollectorDeployed) {
+            error(
+                `Collector: no contract at address ${config.relayHubAddress}`
```

Comment:
We need to change this:
`Collector: no contract at address ${config.relayHubAddress}`

to this:
`Collector: no contract at address ${config. collectorAddress}`